lib/utilities.js

```diff
@@ -54,7 +91,19 @@ function formatDOM(nodes, parentObj, directive) {
         // set the node name if it's not "#text" or "#comment"
         // e.g., "div"
         if (node.nodeName.indexOf('#') !== 0) {
-            nodeObj.name = node.nodeName.toLowerCase();
+            var nodeName = null;
+            // SVG elements are case-sensitive, so correct it if it's wrong
+            for (var j = 0; j < CASE_SENSITIVE_TAGS.length; j++) {
```

Comment:
I don't believe we should correct invalid SVG elements here because I feel it's beyond the scope of what the parser should do and there's a performance cost:
```js
var nodeName = node.nodeName;
// lowercase if element is HTML
if (CASE_SENSITIVE_TAGS.indexOf(nodeName) !== -1) {
  nodeName = nodeName.toLowerCase();
}
nodeObj.name = nodeName;
```

Reply:
This is what I had originally (although `!==` should be `===`), but it doesn't work because the node name is lowercase already, so it's already incorrect by this point and has to be corrected. That said, I can optimize it to remove the loop.

Reply:
Yes, I meant `===` instead of `!==`. Thanks for the catch.